bin/tv-train

```diff
@@ -27,13 +27,17 @@ FLAGS = flags.FLAGS
 
 
 # configure logging
+if 'TV_IS_DEV' in os.environ and os.environ['TV_IS_DEV']:
+    logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s',
+                        level=logging.DEBUG,
+                        stream=sys.stdout)
+else:
+    logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s',
+                        level=logging.INFO,
+                        stream=sys.stdout)
 
-logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s',
-                    level=logging.INFO,
-                    stream=sys.stdout)
 
-
-def _copy_parameters_to_traindir(input_file, target_name, target_dir):
+def _copy_parameters_to_traindir(hypes, input_file, target_name, target_dir):
```

Comment:
Isn't `input_file` also in `hypes`? Shouldn't we - now that we change the function signature anyway - make it simplyer then?